tests/infra/e2e_args.py

```diff
@@ -138,9 +138,9 @@ def cli_args(add=lambda x: None, parser=None, accept_unknown=False):
     )
     parser.add_argument(
         "--join-timer",
-        help="Timer period when tyring to join an existing network (ms)",
+        help="Timer period when trying to join an existing network (ms)",
         type=int,
-        default=4000,
+        default=1000,
```

Comment:
The default in `host/main.cpp` is 1000, so that should be good enough for Python too? A separate concern is that we're still duplicating these args and defaults in both places, but I don't see an easy way to avoid that when so many are actually used in the Python infra.

Reply:
One way would be to call cchost once with —help and collect the defaults from there. Another would be to have both Python and CLI11 fetch them from a config somewhere...

Reply:
This was originally set to 4 secs in https://github.com/microsoft/CCF/pull/680, because the primary would get overloaded with join requests. 1 second should be fine for `cchost` (users of CCF in the wild) but I think we still want to 4 seconds in our tests? Unless something has changed since #680?

Reply:
I've restored the 4s on this and added a comment explaining why.

I think parsing the defaults from `cchost` makes sense and loading from a separate config seems cumbersome (its a third source of the available argument names, in a custom format, with a necessary binding to the C++ expected names at least?). CLI11 allows custom help formatters so we could make one that's more reliably/easily parseable. We could also go further and try to write a formatter that produced JSON rather than string (ie - machine-parseable help formatting), but that's a lot of work for a minor inconvenience.